doc/anchors.nix

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Could this file be auto-generated? And conflicts checked?

Reply:
I think so!

Reply:
Also do not forget about the [bracketed spans](https://pandoc.org/MANUAL.html#extension-bracketed_spans) (used in GNOME docs). Unfortunately, those are not expressible in regex very nicely.

Reply:
It might be sufficient to look for all the instances of `{#sec-name}`.

Reply:
Auto generated anchor rewriting, and bracketed span rewriting is working!